src/containers/ProjectContainer/ProjectContainer.js

```diff
@@ -3,24 +3,40 @@ import ProjectCard from '../ProjectCard/ProjectCard';
 import { connect } from 'react-redux';
 
 export const ProjectContainer = (props) => {
-  const projectCards = props.projects.map(project => {
-    const projectPalettes = props.palettes.filter(palette => palette.project_id === project.id);
-    return <ProjectCard projectTitle={project.name} palettes={projectPalettes} id={project.id} key={project.id}/>
-  });
+  const { searching, search, projects, palettes, loading, error } = props;
+  let projectCards;
+
+  if (searching && search.length) {
```

Comment:
I really like the way you are handling this in redux store and in the conditional here.